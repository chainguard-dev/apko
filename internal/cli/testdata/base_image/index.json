{"schemaVersion":2,"mediaType":"application/vnd.oci.image.index.v1+json","manifests":[{"mediaType":"application/vnd.oci.image.manifest.v1+json","size":402,"digest":"sha256:2ef91a9967f2e1759ea49a8c01cf6a45dd9f9af71fe09bcf2b86175bc4a71314","platform":{"architecture":"amd64","os":"linux"}},{"mediaType":"application/vnd.oci.image.manifest.v1+json","size":402,"digest":"sha256:5a99438a9ced8193f1d71209d0b558fdc0b184aee5cf258e5f7aa9a6ab0f0671","platform":{"architecture":"arm64","os":"linux"}}]}