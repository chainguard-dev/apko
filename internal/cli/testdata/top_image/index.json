{"schemaVersion":2,"mediaType":"application/vnd.oci.image.index.v1+json","manifests":[{"mediaType":"application/vnd.oci.image.manifest.v1+json","size":557,"digest":"sha256:9c4e08d7a6ddc5ffe7bedfe52f1d8c344eace518f8a8d4ddcb231f191e558566","platform":{"architecture":"amd64","os":"linux"}},{"mediaType":"application/vnd.oci.image.manifest.v1+json","size":557,"digest":"sha256:187fd84e9e1772f5e76bd7c4f4c29a884885ece66514a9a3b7fc7a541fb11dd1","platform":{"architecture":"arm64","os":"linux"}}]}