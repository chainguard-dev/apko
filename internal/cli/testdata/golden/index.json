{"schemaVersion":2,"mediaType":"application/vnd.oci.image.index.v1+json","manifests":[{"mediaType":"application/vnd.oci.image.manifest.v1+json","size":476,"digest":"sha256:0c3a3431cb3d08f1de9b3ebd19671fe54c227a6cfd33c1164cb8f67a86baadea","platform":{"architecture":"amd64","os":"linux"}},{"mediaType":"application/vnd.oci.image.manifest.v1+json","size":476,"digest":"sha256:2daa2a0b7c784bbda06b9a342d2b70e72d4f213642f488250fdb62ccd65a4ba7","platform":{"architecture":"arm64","os":"linux"}}],"annotations":{"org.opencontainers.image.created":"1970-01-01T00:00:00Z"}}